### DISCUSSION TITLE
You must be logged in to the server (Unauthorized) while running kubectl after kubeconfig configuration on bootstrapped cluster

### DISCUSSION THREAD

**Discussion by abrekhov** [deckhouse/deckhouse#4198] [Q&A]
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/deckhouse/deckhouse/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/deckhouse/deckhouse/issues) for an issue that matches the one I want to file, without success.

### Version

v1.43.8

### Expected Behavior

kubectl get po -A must work after kubeconfig configuration instruction.

### Actual Behavior

```bash
k get po
error: You must be logged in to the server (Unauthorized)
```

### Steps To Reproduce

1. I used Yandex Cloud provider
2. Use default DOMAIN template
3. Run dhctl with your params
4. After bootstap reconfigure DOMAIN_TEMPLATE with your own domain with wildcard (*.your-company.best) using instructions from website
5. Go to kubeconfig.your-company.best
6. Login with creds
7. kubectl config user/cluster/context using instructions for MacOS

**Comment by konstantin-axenov** [comment]
Can you provide kube-apiserver logs? There have to be an error when you run `k get po`
```
kubectl -n kube-system logs -l component=kube-apiserver -c kube-apiserver
```
